api/src/main/java/org/pmiops/workbench/api/CohortBuilderController.java

```diff
@@ -90,13 +92,13 @@
             /*  Generate the actual query that will grab all the subjects based on criteria.
                 TODO: modifier handling
              */
-            query = generator.handleSearch(request.getType(), params);
+            query = generator.handleSearch(item.getType(), params);
             result = executeQuery(query);
             resultMapper = getResultMapper(result);
             for (List<FieldValue> row : result.iterateAll()) {
-                final Subject subject = new Subject();
-                subject.setVal(row.get(resultMapper.get("val")).getStringValue());
-                subjectSet.addItemsItem(subject);
+                final String subject;
+                subject = row.get(resultMapper.get("val")).getStringValue();
```

Comment:
can initialize and set object in one line like so:
String subject = row.get(resultMapper.get("val")).getStringValue();